packages/synapse-react-client/src/components/RecentPublicationsGrid/RecentPublicationsGrid.tsx

```diff
@@ -178,14 +205,23 @@ function RecentPublicationsGrid(props: RecentPublicationsGridProps) {
               sx={{
                 whiteSpace: 'nowrap',
                 alignSelf: 'flex-start',
+                boxShadow: 'none',
+                padding: '6px 24px',
+                fontWeight: 600,
+                '&:hover': {
+                  boxShadow: 'none',
+                  backgroundColor: '#1A8975',
+                  animationTimingFunction: 'ease-out',
+                  animationDuration: '64ms',
+                },
```

Comment:
Do we really need to change the backgroundColor & animation here? This feels like a change that should apply to all buttons or no buttons, but maybe I am missing context.

Reply:
This wasn't in the linked jira ticket but Adam asked if we could apply these styles to the CTA button but I noticed I put it in the wrong place and made it a override for the ELITE portal instead. Adam also mentioned color-mix() might be a neat option to make it reusable accross portals. I also changed this to use MUI's darken instead of the specific color.